src/styles/catalog.scss

```diff
@@ -0,0 +1,130 @@
+
+.cards {
+  display: grid;
+  grid-template-columns: repeat(1, $size-card-item);
+  justify-content: center;
+  gap: 46px 48px;
+  padding: 50px 48px;
+
+  &__title {
+    color: $cost-color;
+    height: 36px;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+
+    margin: 40px 16px 0;
+  }
+
+  &__img {
+    object-fit: contain;
+    display: block;
+    width: 160px;
+    height: 134px;
+    padding: 32px 19px 0;
+  }
+
+  &__product-code {
+    color:  $price-color;
+    font-size: 10px;
+    line-height: 14px;
+    display: block;
+
+    margin: 0;
+    padding-left: 16px;
+    padding-top: 4px;
+  }
+
+  @media (min-width: 488px) {
+    grid-template-columns: repeat(2, $size-card-item);
+  }
+
+  @media (min-width: 768px) {
+    grid-template-columns: repeat(3, $size-card-item);
+  }
+
+  @media (min-width: 1024px) {
+    grid-template-columns: repeat(4, $size-card-item);
+  }
+
+  &__item {
+    width: $size-card-item;
+    box-sizing: border-box;
+    display: block;
+    margin: 0 auto;
+    background-color: $background-color-main;
+    border-radius: 5px;
+    border: 1px solid #F3F3F3;
+    padding-bottom: 16px;
+    transition: transform 0.3s;
+
+    &:hover {
+      transform: scale(1.2);
+
+      & .card__title {
+        color: #34568b;
+      }
+    }
+  }
+
+  &__rang {
+    display: flex;
+    width: 166px;
```

Comment:
![2023-12-18_16h15_37](https://github.com/mate-academy/layout_catalog/assets/61904995/13bb82a3-f5b8-4d60-b1ab-6235e7ac2df4)
Not fixed. It's redudant style at all - remove it
https://github.com/mate-academy/layout_catalog/assets/61904995/fe3135d9-8b1d-4313-b08f-36cb8b2f1028

Reply:
u7505